UI/obs-app.cpp

```diff
@@ -2110,6 +2110,22 @@ static int run_program(fstream &logFile, int argc, char *argv[])
 		OBSTranslator translator;
 		program.installTranslator(&translator);
 
+		if (!created_log) {
+			create_log_file(logFile);
+			created_log = true;
+		}
+
+#ifdef __APPLE__
```

Comment:
Does this actually need to occur before `if (!already_running)`?

Reply:
We could probably move it further down if that's desirable

Reply:
Good point, moved it accordingly.